inlong-manager/manager-schedule/src/main/java/org/apache/inlong/manager/schedule/airflow/config/AirflowConfig.java

```diff
@@ -68,6 +72,15 @@ public class AirflowConfig extends ClientConfiguration {
     @Value("${schedule.engine.airflow.baseUrl:http://localhost:8080/}")
     private String baseUrl;
 
+    @PostConstruct
+    public void init() {
+        if (!StringUtil.isEmpty(inlongManagerUrl)) {
+            String[] urlInfo = inlongManagerUrl.split(InlongConstants.COLON);
```

Comment:
Need to check the URL, otherwise NPE will occur when the `urlInfo.length` is not equal 2.

Reply:
done.